app/helpers/inquirer.js

```diff
@@ -93,5 +93,26 @@ async function createNewTask(taskController, user) {
     }
 }
 
+async function showFinishedTasks(taskController, user) {
+    let exit = false;
+    const tasksArray = await taskController.getFinishedTasks();
+    const tasksByUser = getTasksByUser(tasksArray, user);
+    const choices = getTaskTitlesMenuChoices(tasksByUser, 'Select a task to view more details:');
+
+    while (!exit) {
+        const menuOption = await inquirer.prompt([{
```

Comment:
You dont need to create a new Object here.. just pass the object created before for inquirer
`const choices`

Reply:
Modified it and passed the object "menu"